constants.py

```diff
@@ -50,7 +51,7 @@
         ]
 
 # The trees for which we save binary packages.
-releaseBranches = ["bb-*-release", "preview-10.*"]
+releaseBranches = ["bb-*-release", "preview-10.*", "preview-11.*"]
```

Comment:
do you remember if there is any reason for `preview-10-*` instead of `preview-*`?

Reply:
`preview-*` does seem like a better idea.